doc/src/cards/ScenarioNode.mdx

```diff
@@ -5,99 +5,130 @@ menu: Cards
 ---
 
 import { Playground, Props } from 'docz';
+import { useState } from 'react';
 import { ScenarioNode } from '@cosmotech/ui';
 
 # ScenarioNode
 
 The aim of this component is to easily display a short scenario's description.
 
-### Props
+## Props
 
 <Props of={ScenarioNode} />
 
-### Basic Usage with scenario status 'Created'
+## Basic Usage with scenario status 'Created'
 
 <Playground>
-  <ScenarioNode
-    datasets={[
-      {
-        id: 'D-000001',
-        name: 'A wonderful dataset name',
-        tags: ['dataset'],
-      },
-      {
-        id: 'D-000002',
-        name: 'Second wonderful dataset name',
-        tags: ['dataset'],
-      },
-    ]}
-    scenario={{
-      id: 's-00001',
-      name: 'Value 1',
-      parentId: null,
-      creationDate: '2021-09-22T15:53:22.622797996Z',
-      datasetList: ['D-000001'],
-      ownerName: 'Bob Bobinson',
-      state: 'Created',
-    }}
-    deleteScenario={() => alert('Scenario deletion')}
-  />
+  {() => {
+    const [open, setOpen] = useState(true);
+    return (
+      <ScenarioNode
+        datasets={[
+          {
+            id: 'D-000001',
+            name: 'A wonderful dataset name',
+            tags: ['dataset'],
+          },
+          {
+            id: 'D-000002',
+            name: 'Second wonderful dataset name',
+            tags: ['dataset'],
+          },
+        ]}
+        scenario={{
+          id: 's-00001',
+          name: 'Value 1',
+          parentId: null,
+          creationDate: '2021-09-22T15:53:22.622797996Z',
+          datasetList: ['D-000001'],
+          ownerName: 'Bob Bobinson',
+          state: 'Created',
+        }}
+        showDeleteIcon={true}
+        isExpanded={open}
+        setIsExpanded={(newIsExpanded) => setOpen(newIsExpanded)}
+        deleteScenario={() => alert('Scenario deleted')}
+        onScenarioRename={(newScenarioName) => alert('Scenario renamed')}
```

Comment:
We don't need `newScenarioName` argument in this function :slightly_smiling_face: